packages/theme/src/manager.ts

```diff
@@ -36,19 +36,10 @@ export class ThemeManager extends LabThemeManager {
         reject(`Stylesheet failed to load: ${href}`);
       });
 
-      this._link = link;
-      this._unloadCSS();
       document.body.appendChild(link);
+      this['_links'].push(link);
```

Comment:
Not the most elegant since we are extending `LabThemeManager` from upstream, and `_links` is managed as a private attribute.

Reply:
As a follow-up, we should check whether we can just reuse the upstream plugin instead of having our own here.

Probably this was to be able to define the custom themes url, but maybe that is already configurable.